liquidcore/home/models.py

```diff
@@ -0,0 +1,13 @@
+import string
+from django.contrib.auth.models import AbstractUser
+
+USERNAME_CHARS = string.ascii_letters + string.digits + '.'
+
+
+class User(AbstractUser):
+    class Meta:
+        db_table = 'auth_user'
+
+    def save(self, *args, **kwargs):
```

Comment:
Do we really have to change the user model? Is there no easier way to enforce this? Maybe as a pre-save hook?

Reply:
On a second look through the docs,

> [All built-in signals are sent using the send() method.](https://docs.djangoproject.com/en/2.2/ref/signals/)

> [send() does not catch any exceptions raised by receivers; it simply allows errors to propagate](https://docs.djangoproject.com/en/2.2/topics/signals/#django.dispatch.Signal.send)

Yes, we should be able to throw in the signal handler and expect the save will be aborted.